sw/airborne/subsystems/ahrs/ahrs_float_cmpl.c

```diff
@@ -208,324 +208,325 @@ void ahrs_fc_update_accel(struct FloatVect3 *accel, float dt)
      */
-      {0.0, 0.0, -ahrs_fc.ltp_vel_norm);
+    {
+      0.0, 0.0, -ahrs_fc.ltp_vel_norm);
 #else
     /* assume tangential velocity along body x-axis */
-      {ahrs_fc.ltp_vel_norm, 0.0, 0.0};
+    {ahrs_fc.ltp_vel_norm, 0.0, 0.0};
 #endif
-    struct FloatRMat *body_to_imu_rmat = orientationGetRMat_f(&ahrs_fc.body_to_imu);
-    struct FloatRates body_rate;
-    float_rmat_transp_ratemult(&body_rate, body_to_imu_rmat, &ahrs_fc.imu_rate);
-    struct FloatVect3 acc_c_body;
-    VECT3_RATES_CROSS_VECT3(acc_c_body, body_rate, vel_tangential_body);
+      struct FloatRMat *body_to_imu_rmat = orientationGetRMat_f(&ahrs_fc.body_to_imu);
+      struct FloatRates body_rate;
+      float_rmat_transp_ratemult(&body_rate, body_to_imu_rmat, &ahrs_fc.imu_rate);
+      struct FloatVect3 acc_c_body;
+      VECT3_RATES_CROSS_VECT3(acc_c_body, body_rate, vel_tangential_body);
 
-    /* convert centrifugal acceleration from body to imu frame */
-    struct FloatVect3 acc_c_imu;
-    float_rmat_vmult(&acc_c_imu, body_to_imu_rmat, &acc_c_body);
+      /* convert centrifugal acceleration from body to imu frame */
+      struct FloatVect3 acc_c_imu;
+      float_rmat_vmult(&acc_c_imu, body_to_imu_rmat, &acc_c_body);
 
-    /* and subtract it from imu measurement to get a corrected measurement of the gravity vector */
-    VECT3_DIFF(pseudo_gravity_measurement, imu_accel, acc_c_imu);
+      /* and subtract it from imu measurement to get a corrected measurement of the gravity vector */
+      VECT3_DIFF(pseudo_gravity_measurement, imu_accel, acc_c_imu);
 
-  } else {
-    VECT3_COPY(pseudo_gravity_measurement, imu_accel);
-  }
+    } else {
+      VECT3_COPY(pseudo_gravity_measurement, imu_accel);
+    }
 
-  VECT3_CROSS_PRODUCT(residual, pseudo_gravity_measurement, c2);
+    VECT3_CROSS_PRODUCT(residual, pseudo_gravity_measurement, c2);
 
-  /* FIR filtered pseudo_gravity_measurement */
+    /* FIR filtered pseudo_gravity_measurement */
 #define FIR_FILTER_SIZE 8
-  static struct FloatVect3 filtered_gravity_measurement = {0., 0., 0.};
-  VECT3_SMUL(filtered_gravity_measurement, filtered_gravity_measurement, FIR_FILTER_SIZE - 1);
-  VECT3_ADD(filtered_gravity_measurement, pseudo_gravity_measurement);
-  VECT3_SDIV(filtered_gravity_measurement, filtered_gravity_measurement, FIR_FILTER_SIZE);
-
-  if (ahrs_fc.gravity_heuristic_factor) {
-    /* heuristic on acceleration (gravity estimate) norm */
-    /* Factor how strongly to change the weight.
-     * e.g. for gravity_heuristic_factor 30:
-     * <0.66G = 0, 1G = 1.0, >1.33G = 0
-     */
-
-    const float g_meas_norm = float_vect3_norm(&filtered_gravity_measurement) / 9.81;
-    ahrs_fc.weight = 1.0 - ahrs_fc.gravity_heuristic_factor * fabs(1.0 - g_meas_norm) / 10.0;
-    Bound(ahrs_fc.weight, 0.15, 1.0);
-  } else {
-    ahrs_fc.weight = 1.0;
-  }
+    static struct FloatVect3 filtered_gravity_measurement = {0., 0., 0.};
```

Comment:
isn't pprz style with 2 spaces? Pls run the stylecheck script on this.

Reply:
Actually, all these changes were done by fix_code_style

Reply:
If I run it again, it doesn't change it. It does give some weird error warning though:
~/paparazzi$./fix_code_style.sh sw/airborne/subsystems/ahrs/ahrs_float_cmpl.c
Using astyle version 2.05.1
(standard_in) 1: syntax error
./fix_code_style.sh: line 18: [: -eq: unary operator expected
Unchanged  sw/airborne/subsystems/ahrs/ahrs_float_cmpl.c

Reply:
Don't know why the script does the warning, maybe @flixr knows more?

However, the file broke the style check - compare lines 206 `const struct FloatVect3 vel_tangential_body =` before and after. Clearly the correct style is before applying the style check. 

Pls revert and check your changes manually. 

Also there is a typo at line 212 - the end bracket should be `}` instead of `)`